components/camel-kafka/src/main/java/org/apache/camel/component/kafka/KafkaConfiguration.java

```diff
@@ -62,6 +62,10 @@
     @UriParam
     private String clientId;
 
+    //interceptor.classes
+    @UriParam
```

Comment:
Can you add label = "monitoring" as its kinda intended for this kind of things, isnt it?